### ISSUE TITLE
Deprecation Notice: Creation of dynamic property

### ISSUE THREAD

**Issue by Starraider** [eibiflo/cf_cookiemanager#45]
In TYPO3 v12, PHP v8.2.24 I get this error message in the backend:
```
PHP Runtime Deprecation Notice: Creation of dynamic property CodingFreaks\CfCookiemanager\Domain\Model\Scans::$foundProvider is deprecated in /releases/12/vendor/codingfreaks/cf-cookiemanager/Classes/Domain/Repository/ScansRepository.php line 130
```

**Comment by eibiflo** [comment]
Hey @Starraider,  

the issue should be fixed now. 
Thanks for reporting it. 


Fix will be available in version 1.7.2.
If you need it immediately, you can install the dev branch for now. 
[See changelog](https://docs.typo3.org/p/codingfreaks/cf-cookiemanager/main/en-us/ChangeLog/Index.html)

**Comment by eibiflo** [comment]
Closed with Release 1.7.2
